modules/core/packet-server/types/events.go

```diff
@@ -0,0 +1,18 @@
+package types
+
+import (
+	"fmt"
+
+	ibcexported "github.com/cosmos/ibc-go/v9/modules/core/exported"
+)
+
+// IBC channel events
+const (
+	AttributeKeyChannelID = "channel_id"
+)
+// IBC channel events vars
+var (
+	EventTypeCreateChannel = "create_channel"
```

Comment:
any reason this is not a constant?

Reply:
feel like a const makes more sense here, we can worry about it when we move stuff over to msg_server_v2_land anyway